src/accelerate/commands/config/cluster.py

```diff
@@ -129,8 +130,9 @@ def get_cluster_input():
     else:
         dynamo_backend = DynamoBackend.NO
 
+    use_mps = not use_cpu and is_mps_available()
     deepspeed_config = {}
-    if distributed_type in [DistributedType.MULTI_GPU, DistributedType.NO]:
+    if distributed_type in [DistributedType.MULTI_GPU, DistributedType.NO] and not use_mps:
```

Comment:
It also makes sense if we keep `MPS` to make this logic here the same. It adds some more complexity in the refactor/improvement I think